scalableminds/webknossos
# Isosurfaces for volume annotation layers with fallback segmentation

### URL of deployed dev instance (used for testing):
- https://isosurfacefallback.webknossos.xyz

### Steps to test:
- Create volume annotation with fallback layer (default for datasets with segmentation layers)
- Pick an existing segment’s color, brush some (brush in multiple z-layers, so that isosurface subsampling won’t skip over it)
- right click segment, select ad-hoc mesh, wait, watch 3d viewport
- full segment’s isosurface should appear, featuring the brushed changes.
- also validate that isosurface loading in dataset view mode still works
- also validate that isosurface loading for volume annotations _without_ fallback segmentation still works

### Issues:
- fixes #4930 

------
